recipe/0007-pycryptosat-fix-install-path.patch

```diff
@@ -0,0 +1,11 @@
+--- python/CMakeLists.txt
++++ python/CMakeLists.txt
+@@ -61,7 +61,7 @@
+                   ALL DEPENDS ${OUTPUT}/timestamp)
+ 
+ install(CODE "execute_process(
+-    COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} install --root=\$ENV{DESTDIR}/ --prefix=${CMAKE_INSTALL_PREFIX} --record files.txt
++    COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} install --record files.txt
```

Comment:
I always forget why, but for Conda (due to some historical reason) we have an additional `%PREFIX%/Library` directory on Windows. Hence, we normally have `CMAKE_INSTALL_PREFIX` point to that directory. On the other hand, the Python libraries live outside of that at `%PREFIX%/Lib` so that we can't use `--prefix=${CMAKE_INSTALL_PREFIX}` here. But omitting `--prefix` and `--root` altogether works for us here since it then picks up the standard location we want it to install to. (And I'm not sure what that `--root=\$ENV{DESTDIR}/` does here. No idea where that `DESTDIR` points to. Usually `--prefix` should suffice (at least when `CMAKE_INSTALL_PREFIX` is an absolute path, I guess).)